src/concat.js

```diff
@@ -13,8 +13,8 @@ var invoker = require('./invoker');
  * @category List
  * @sig [a] -> [a] -> [a]
  * @sig String -> String -> String
- * @param {Array|String} a
- * @param {Array|String} b
+ * @param {Array|String|Object} a
+ * @param {Array|String|Object} b
```

Comment:
Yeah, because of this https://github.com/ramda/ramda/blob/7e25f1f6e75cb55d9653ebb6ba46e415e2edcff3/test/concat.js#L33-L35.

Do you think it's a bad idea?